pydis_site/templates/home/index.html

```diff
@@ -19,21 +19,21 @@ <h1 class="is-size-1">Who are we?</h1>
         <div class="column is-half">
           <p>
             We're a large community focused around the Python programming language.
-            We believe anyone can learn programming, and are very dedicated to helping
-            novice developers take their first steps into the world of code. We also
+            We believe anyone can learn to code, and are very dedicated to helping
+            novice developers take their first steps into the world of programming. We also
             attract a lot of expert developers who are seeking friendships, collaborators,
-            or looking to hone their craft by teaching and getting involved in the community.
+            or to hone their craft by teaching and getting involved with the community.
```

Comment:
Having a list which uses two nouns as objects of the sentence and then switches to an infinitive of a verb reads awkwardly. Not sure how to rephrase it though.

Reply:
added some more words here to hopefully make it flow better.